cmd/charm/charmcmd/whoami_test.go

```diff
@@ -85,7 +85,7 @@ func (s *whoamiSuite) TestLoggedIn(c *gc.C) {
 	stdout, stderr, exitCode := run(c.MkDir(), "whoami")
 	c.Assert(stderr, gc.Equals, "")
 	c.Assert(exitCode, gc.Equals, 0)
-	c.Assert(stdout, gc.Equals, "test-user\ntest-group1 test-group2\n")
+	c.Assert(stdout, gc.Equals, "User: test-user\nGroup membership: test-group1, test-group2\n")
```

Comment:
Can we make it clear that groups are ordered alphabetically in the test?
Could you also add tests for other --format options?